conv/conv.tex

```diff
@@ -0,0 +1,116 @@
+\documentclass[utf8x,hyperref={pdfpagelabels=false}]{beamer}
+
+\usepackage[utf8x]{inputenc}
+\usepackage[OT1]{fontenc}
+\usepackage{graphicx}
+\usepackage{xcolor}
+\usepackage{amsmath}
+
+\usetheme{Malmoe}  % Now it's a beamer presentation with the lisa theme!        
+\usecolortheme{beaver}
+\setbeamertemplate{footline}[page number]
+\setbeamertemplate{navigation symbols}{}
+
+\title{Convolutions for neural networks}
+
+\author{%
+Arnaud Bergeron
+}
+\date{August 10, 2015}
+
+\begin{document}
+
+\begin{frame}[plain]
+ \titlepage
+\end{frame}
+
+\setcounter{page}{1}
+
+\section{Convolution}
+
+% Definitions
+\begin{frame}
+Some vocabulary:
+\begin{description}[feature maps]
+\item[feature map] analogue to layer (the activations, not the weights) for a neural network.
```

Comment:
For me, "feature map" was equivalent to "channel". This is why the documentation of `nnet.conv2d` talks about "a stack of feature maps".